Desktop/components/JASP/Widgets/Supporters.qml

```diff
@@ -0,0 +1,104 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import JASP.Widgets
+
+
+Item
+{
+	height:							supportColumn.height
+	property int fontPixelSize:		14
+
+	ColumnLayout
+	{
+		id:							supportColumn
+		spacing:					welcomeRoot.scaler * 20
+		anchors.left:				parent.left
+		anchors.right:				parent.right
+
+		Text
+		{
+			text:					qsTr("JASP is supported by the following institutions:").replace(/, /g, ",&nbsp;").replace(/\n\n/g, "<br><br>")
+			font.family:			jaspTheme.font
+			font.weight:			Font.Normal
+			font.pixelSize:         fontPixelSize * welcomeRoot.scaler
+			color:					jaspTheme.white
+			wrapMode:				TextEdit.Wrap
+			renderType:				Text.QtRendering
+			textFormat:				Text.StyledText
+			horizontalAlignment:	Text.AlignHCenter
+			Layout.fillWidth:		true
+		}
+
+		SwipeView
+		{
+			id:						swipeSupporters
+			orientation:			Qt.Vertical
+			Layout.fillWidth:		true
+			clip:					true
+			currentIndex:			Math.floor(Math.random() * count)
+			wheelEnabled:			true
+
+			Timer
+			{
+				interval:			5000
+				repeat:				true
+				running:			true
+				onTriggered:		{ swipeSupporters.setCurrentIndex((swipeSupporters.currentIndex + 1) % swipeSupporters.count) }
+			}
+
+			Repeater
+			{
+				model:
+				[
+					"Tilburg University",
+					"KU Leuven",
+					"Nyenrode Business University",
+					"M&S / Faculty of Social Sciences / Utrecht University",
+					"University of Amsterdam"
+				]
+
+				Text
+				{
+					text:					String(modelData).replace(/&/g, "&amp;").replace(/, /g, ",&nbsp;").replace(/\n/g, "<br>")
+					font.family:			jaspTheme.font
+					font.weight:			Font.Bold
+					font.pixelSize:         (4 + fontPixelSize) * welcomeRoot.scaler
+					color:					jaspTheme.white
+					wrapMode:				TextEdit.Wrap
+					renderType:				Text.QtRendering
+					textFormat:				Text.StyledText
+					horizontalAlignment:	Text.AlignHCenter
+
+					width:					swipeSupporters.width
+				}
+			}
+
+
+		}
+
+		Text
+		{
+			text:					qsTr("Suggest your institution joins the JASP Cooperative").replace(/, /g, ",&nbsp;").replace(/\n\n/g, "<br><br>")
```

Comment:
Is this sentence from EJ?? My english is not so good, but I don't understand it either... And JASP is not a cooperative.

Reply:
Im not making these texts up :p

Reply:
it's a cooperative in the broad sense of the word ^^